src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,20 @@ public partial struct OSPlatform : System.IEquatable<System.Runtime.InteropServi
         public static bool operator !=(System.Runtime.InteropServices.OSPlatform left, System.Runtime.InteropServices.OSPlatform right) { return default(bool); }
         public override string ToString() { return default(string); }
     }
+
+    public enum Architecture
+    {
+        x86,
+        x64,
+        ARM
```

Comment:
```Arm``` is the brand name for Arm 32 bit, and when we add Arm64, it will be whatever it's branded as.